packages/server/graphql/mutations/helpers/getCCFromCustomer.ts

```diff
@@ -1,12 +1,19 @@
 import Stripe from 'stripe'
+import {getStripeManager} from '../../../utils/stripe'
+
+export default async function getCCFromCustomer(customer: Stripe.Customer) {
+  if (typeof customer.default_source !== 'string') {
+    throw new Error(`Could not find default_source for ${customer.id}`)
+  }
+  const manager = getStripeManager()
+  const card = await manager.retrieveSource(customer.id, customer.default_source)
```

Comment:
-2 now that this is async we'll need to fix the following:
 https://github.com/ParabolInc/parabol/blob/4172e4446fb0b6206fa585a36d037476899e6330/packages/server/graphql/private/mutations/stripeUpdateCreditCard.ts

more importantly-- why didn't typescript tell us about this error?! could you run `yarn typecheck` in the server package & triage why?

Reply:
How strange! I ran `yarn typecheck` in the server package, but there were no complaints. I played around with the config but didn't get any change in behaviour

Reply:
issue created for debugging: https://github.com/ParabolInc/parabol/issues/6990